### ISSUE TITLE
an advise

### ISSUE THREAD

**Issue by yitony** [openresty/lua-resty-limit-traffic#8]
on line 144 of req.lua 

    142     rec_cdata.excess = excess
    143     rec_cdata.last = rec.last
    144     dict:set(key, ffi_str(rec_cdata, rec_size))
    145     return true

I think it will save mem if you set a expire time on the key, like below 
dict:set(key, ffi_str(rec_cdata, rec_size),60)

same advise  on con.lua ,bug you need to change add to set

**Comment by agentzh** [comment]
@yitony Yes, setting an exptime helps saving shm storage in some cases. Will you create a pull request for that? Thanks!

Regarding the `uncommit` method, it is documented (well, I've just fixed a typo in the docs and added some more stuff):

https://github.com/openresty/lua-resty-limit-traffic/blob/master/lib/resty/limit/req.md#uncommit
